app/src/main/java/org/mozilla/fenix/components/metrics/Metrics.kt

```diff
@@ -201,7 +204,21 @@ sealed class Event {
     object TabsTrayShareAllTabsPressed : Event()
     object TabsTrayCloseAllTabsPressed : Event()
 
+    object ProgressiveWebAppOpenFromHomescreenTap : Event()
+    object ProgressiveWebAppInstallAsShortcut : Event()
+
     // Interaction events with extras
+
+    data class ProgressiveWebAppForeground(val timeOnForeground: Long) : Event() {
```

Comment:
Nit: this is a little confusing - it sounds a little like "time spent on foreground" rather than "time foregrounded"